CHANGELOG.md

```diff
@@ -19,12 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
```

Comment:
You're missing a video for bullet hell access: https://www.youtube.com/watch?v=yDjHOQ-aG1w&list=PL3l1MUvtY68XxNORgBQs--5vXjmcnpJ2m&index=35
Also speed boosting challenge: https://www.youtube.com/watch?v=VLu8ujljuGg&list=PL3l1MUvtY68XxNORgBQs--5vXjmcnpJ2m&index=36